js/homeview.js

```diff
@@ -456,6 +456,13 @@ enyo.kind({
 		this.currentView = newView;
 		stats.trace(constant.viewNames[oldView], 'change_view', constant.viewNames[newView]);
 
+		if (
+			(constant.viewNames[oldView] == 'list_view' && constant.viewNames[newView] == 'home_view') || 
+			(constant.viewNames[oldView] == 'list_view' && constant.viewNames[newView] == 'neighborhood_view')
+		) {
+			sessionStorage.setItem("scroll-value", scrollbar_position());
```

Comment:
Not a big fan of using `sessionStorage`. A context storage class is available in settings class: https://github.com/llaske/sugarizer/blob/dev/lib/settings.js